engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -86,8 +87,17 @@ angular.module('Bastion.capsule-content').controller('CapsuleContentController',
                     $scope.syncTask = aggregateTasks(syncStatus['active_sync_tasks']);
 
                 } else if (syncStatus['last_failed_sync_tasks'].length > 0) {
+                    errorCount = $scope.syncTask.humanized.errors.length;
                     $scope.syncTask = pickLastTask(syncStatus['last_failed_sync_tasks']);
-                    $scope.syncErrorMessages = $scope.syncTask.humanized.errors;
+                    if (errorCount > 0) {
+                        $scope.syncErrorMessages = [$scope.syncTask.humanized.errors[0]];
+                    }
+
+                    if (errorCount > 2) {
```

Comment:
I honestly like it better this way as it draws more attention.  On this page it makes more sense than i would say it may on other pages.  But honestly not worth arguing :)

Updated